packages/oasis-actions/src/protocols/aave/getAaveProtocolData.ts

```diff
@@ -172,3 +187,26 @@ async function getAaveV3ProtocolData({
 }
 
 export type AaveProtocolData = ReturnType<typeof getAaveProtocolData>
+
+type AllowedContractNames = 'poolDataProvider' | 'pool' | 'aaveOracle'
+async function getAbiForContract(contractName: AllowedContractNames, provider: providers.Provider) {
+  const network = await getForkedNetwork(provider as providers.JsonRpcProvider)
```

Comment:
This is mesleading. If we operation on mainnet or optimism or any network not through hardhat... it's not a forked network.